_apps/gpg.md

```diff
@@ -0,0 +1,41 @@
+---
+title: GPG
+description: Enforce GPG signatures on Pull Requests
+slug: probot-gpg
```

Comment:
What would you think about just calling the app `gpg`? We've been trying to use really basic names (like `dco`, `welcome`, `stale`, etc) to set the precedent that these apps should aim to be feature complete.

Reply:
Okay I’ll rename my GitHub App and update this in a bit.